compat/src/Children.js

```diff
@@ -16,11 +13,7 @@ export const Children = {
 		return children ? toChildArray(children).length : 0;
 	},
 	only(children) {
-		children = toChildArray(children);
-		if (children.length !== 1) {
-			throw new Error('Children.only() expects only one child.');
```

Comment:
Iirc we had a bug report to add this specific error to match React's behavior.

Reply:
Stumbled upon that change this morning too. I really vote for keeping this check as React does does this check in production mode so we should do the check in compat too. I guess here the compatibility is more important than the byte wins. Maybe we could add a minified error message to save some bytes?

Reply:
Yeah I was thinking it must have been. Does it need to be the exact same string though?

Reply:
Update: I changed it to throw a string with a very cheap reference to Children.only.